hexrd/cli/fit_grains.py

```diff
@@ -54,6 +55,11 @@ def load(cls, fname):
         """
         return cls(**np.load(fname))
 
+    @classmethod
+    def from_grains_out(cls, fname):
+        """Read hexrd grains output file"""
+        return cls.from_array(np.loadtxt(fname))
```

Comment:
Nice, Don! This class is looking like it is becoming very useful :slightly_smiling_face: